<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pilotageLumiereClass</class>
 <widget class="QMainWindow" name="pilotageLumiereClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pilotageLumiere</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labellumiere1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Lumière 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lumière 2 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lumière 3 :</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere1rouge">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere1vert">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere1bleu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere2rouge">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere2vert">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere2bleu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere3rouge">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere3vert">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="barlumiere3bleu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere1rouge">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Rouge :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere2rouge">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Rouge :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere3rouge">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Rouge :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere1vert">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vert</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere2vert">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vert</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere3vert">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vert</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere1bleu">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Bleu</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere2bleu">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Bleu</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellumiere3bleu">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Bleu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pilotageLumiere.qrc"/>
 </resources>
 <connections/>
</ui>
